### PR TITLE
Create economic-history-review.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/81993/style-request-economic-history-review

Two problems I have for the bibliography:
1. Books: How to remove the dot between title and (publisher-place, year)?
Required output: Church, R., _The history of the British coal industry, 3,1830–1913, Victorian pre-eminence_ (Oxford, 1986).

2. chapters: how to move the chapter pages to the end?
Required output:
Landes, D., ‘Technological change and development in western Europe, 1750–1914’, in H. J. Habakkuk and M. M. Postan, eds., _Cambridge economic history of Europe_, VI, pt. 1, _The industrial revolution and after_ (Cambridge, 1965), pp. 274-601.

### CODE REVIEW

File: economic-history-review.csl [hunk 1]
```diff
@@ -0,0 +1,899 @@
...
+    </sort>
+    <layout suffix=".">
+      <group delimiter=". ">
+        <group delimiter=": ">
+          <group delimiter=", ">
+            <group delimiter=" ">
+              <group delimiter=". ">
+                <group delimiter=" ">
+                  <group delimiter=", ">
+                    <group delimiter=". ">
```
Comment (rmzelle) [inline economic-history-review.csl]: > Books: How to remove the dot between title and (publisher-place, year)?

I think it's the delimiter above.

File: economic-history-review.csl [hunk 2]
```diff
@@ -0,0 +1,899 @@
...
+                      <text macro="description"/>
+                      <text macro="secondary-contributors"/>
+                      <group delimiter=", ">
+                        <text macro="container-contributors"/>
+                        <text macro="container-title"/>
+                      </group>
+                      <text macro="locators-join-with-period"/>
+                    </group>
+                    <text macro="locators-join-with-comma"/>
+                    <text macro="locators-chapter"/>
```
Comment (rmzelle) [inline economic-history-review.csl]: > chapters: how to move the chapter pages to the end?

Chapter pages are generated by this macro call.

File: economic-history-review.csl (hunk 1 above)
Comment (POBrien333) [reply economic-history-review.csl:866]: So, I am aware I can just the delimiter here, but then I'm breaking this whole "chicago" nomenclature of the macros (...join-with-comma/period/space). What's the best way of doing this?

Comment (rmzelle) [reply economic-history-review.csl:866]: 🤷‍♂️

@adam3smith?

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**